requirements.txt

```diff
@@ -3,7 +3,7 @@ h5py
 lxml
 
 # Calculation
-numpy==1.*
+numpy
```

Comment:
Should this be `numpy==2.*` to ensure the user has numpy 2 installed, and not 1 (or a future version)?

Reply:
If numpy versions properly, v1.x releases should never be the 'latest'.

Reply:
No, the changes made here seem to be backward compatible with numpy < 2... `numpy.NaN` used to be an alias for `numpy.nan` in numpy 1.2x, and `numpy.string_` was an alias for `numpy.bytes_`.  This PR just removes the use of those deprecated aliases.  You might want to set a minimum version of 1.20 or something.

Reply:
> If numpy versions properly, v1.x releases should never be the 'latest'.

I was more thinking of if the user had Numpy 1 already installed, it would satisfy this requirement. But if the changes are backwards compatible then that won't be an issue.